<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSimulator</class>
 <widget class="QWidget" name="DeviceSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">#tabWidget{
border:1px solid grey;
}
QLabel{
font: 20px &quot;微软雅黑&quot;;
background-color: rgb(0, 100, 0);
color: rgb(255, 255, 255);
}
QPushButton{
font: 18px &quot;微软雅黑&quot;;
}
</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_card">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QLineEdit" name="cardId">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>卡号</string>
       </property>
      </widget>
      <widget class="QPushButton" name="card_submit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>431</width>
         <height>311</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="card0">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="card1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="card2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="card3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="card4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>0</y>
          <width>111</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>点
击
卡
号
快
捷
写
入</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_rfid">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maxCount">
        <number>16</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>天线序号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>RFID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="rfid">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>221</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>动作</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="rfidIn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>进入范围</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rfidOut">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>80</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>离开范围</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_9">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>180</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_10">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_11">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_12">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="id_13">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>340</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="rfidCheck">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>130</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_server">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
